modules/effects/schematics/ng-add/schema.json

```diff
@@ -27,6 +27,11 @@
       "description": "Flag to indicate if a dir is created."
     },
     "skipTest": {
+      "type": "boolean",
+      "description": "(Deprecated) Use skipTests instead",
```

Comment:
Fyi, instead of adding deprecated to the description we could use the `x-deprecated` property.
This way the cli will log a warning when `skipTest` is being used.

As an example, see the [entryComponent option](https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/component/schema.json#L138)

Reply:
Still needs `x-deprecated`